lib/omnibus/metadata.rb

```diff
@@ -218,7 +218,7 @@ def truncate_platform_version(platform_version, platform)
           # version is the same as Windows 2012R2. It's only here for completeness
           # and documentation.
           when /6\.3\.\d+/, "8.1" then "8.1"
-          when /^10\.0/ then "10"
+          when /^10.*/ then "10"
           else
             raise UnknownPlatformVersion.new(platform, platform_version)
           end
```

Comment:
To avoid making this catch to much, let's instead split it: 

`when "10", /^10\.0/ then "10"`

This way it will match exactly 10, and 10.x, but not "100".